README.md

```diff
@@ -29,7 +29,7 @@ $ bower install --save bchaddrjs
 
 ### Manually
 
-You may also download the distribution file manually and place it within your third-party scripts directory: [dist/bchaddrjs-0.3.2.min.js](https://cdn.rawgit.com/bitcoincashjs/bchaddrjs/master/dist/bchaddrjs-0.3.2.min.js).
+You may also download the distribution file manually and place it within your third-party scripts directory: [dist/bchaddrjs-0.3.3.min.js](https://cdn.rawgit.com/bitcoincashjs/bchaddrjs/master/dist/bchaddrjs-0.3.3.min.js).
```

Comment:
this is not relevant
we are in the fork, the link doesn't make sense, undo readme changes